content/announcements/MiniAsync-v0-1-0-release.md

```diff
@@ -0,0 +1,46 @@
+---
+# News article title
+title: "Miniasync v0.1.0 Release"
+
+# Creation date
+date: 2022-05-20T16:13:23+02:00
+
+# Publish immediately. 
+draft: false
+
+# Hero image
+hero_image: "images/news/my_news_article_hero.jpg"
+
+# Brief description
+description: ""
+
+# Event image
+image: "https://opengraph.githubassets.com/706d2635f355e31fa44b50d0e337af4ce73086e7/pmem/pmdk"
+
+# Announcement category
+announcements: ['pmem']
+
+# Post type
+type: "announcement"
+
+# Featured. Specify true or false to show on homepage
+featured: 
+---
+
+We've just released a first experimental version of miniasyc library. I'd like to thank everyone who contributed.
+
+The library provides the API for asynchronous memory operations
+through the use of features and runtime mechanisms and independence
+in terms of hardware by implementing a virtual data mover.
```

Comment:
`The library provides a framework for the composition and execution of asynchronous tasks in C, as well as an abstract interface for hardware-accelerated data movement.`